ui-guidelines/css/guidelines.css

```diff
@@ -165,8 +159,58 @@ pre {
 	font-size: 1em;
 	font-weight: bold;
 }
-/*==== HELPERS ====*/
-.top-half-extra {margin-top: 2em;}
-.top-extra {margin-top: 4em;}
-.top-low {margin-top: 1.5em;}
-.bottom-half-extra {margin-bottom: 1.5em;}
\ No newline at end of file
+.btn {
+	display: none;
```

Comment:
I see below that the button becomes visible when hovering over parent element. Anyway, it can be a bit misleading if that's a styleguide for buttons.